pkgs/applications/video/gyroflow/default.nix

```diff
@@ -1,48 +1,63 @@
-{ lib, rustPlatform, fetchFromGitHub, callPackage, makeDesktopItem
-, clang, copyDesktopItems, patchelf, pkg-config, wrapQtAppsHook
-, alsa-lib, bash, ffmpeg, mdk-sdk, ocl-icd, opencv, qtbase, qtdeclarative, qtsvg
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeDesktopItem,
+  clang,
+  copyDesktopItems,
+  patchelf,
+  pkg-config,
+  qt6,
```

Comment:
I think this allows to replace `gyroflow = qt6Packages.callPackage` with `gyroflow = callPackage` in `all-packages.nix` (for a later move to `pkgs/by-name`).